ledger-core-tezos/src/CMakeLists.txt

```diff
@@ -87,20 +87,22 @@ if(IS_IOS GREATER_EQUAL 0)
         \${BUILT_PRODUCTS_DIR}/\${PRODUCT_NAME}.framework/\${PRODUCT_NAME}
     )
 else()
-    add_library(ledger-core-tezos-obj OBJECT
-        ${ledger-core-tezos-sources}
-        ${SRC_FILES}
-        ${CORE_SRC_FILES}
-        ${HEADERS_FILES}
-        ${CORE_HEADER_FILES})
+
+    add_library(ledger-core-tezos-api-obj OBJECT ${ledger-core-tezos-api-sources})
+
+    add_library(ledger-core-tezos-obj OBJECT ${SRC_FILES} ${CORE_SRC_FILES} ${CORE_HEADER_FILES})
```

Comment:
We lost `${HEADERS_FILES}` here, I suppose it's just because they are correctly in the `include_directories` of the target, but why do we need to add `${CORE_HEADER_FILES}` then ?